go/vt/vtorc/logic/tablet_discovery.go

```diff
@@ -117,7 +136,10 @@ func OpenTabletDiscovery() <-chan time.Time {
 		log.Error(err)
 	}
 	// Parse --clusters_to_watch into a filter.
-	updateShardsToWatch()
+	err := initializeShardsToWatch()
+	if err != nil {
+		log.Fatalf("Error parsing clusters to watch: %v", err)
```

Comment:
Nit, but IMO it would be nicer to mention `--clusters_to_watch` to help the operator track down what they need to fix (could be in a config file e.g.).

Reply:
Done